packages/core/src/components/slider/slider.stories.tsx

```diff
@@ -254,12 +254,21 @@ const Template = ({
     
      <!-- Script tag for demo purposes -->
     <script>
-      slider = document.querySelector('tds-slider')
+      const slider = document.querySelector('tds-slider');
```

Comment:
this will error in the console if you navigate from the docs and back, can't use const in the code preview since we don't know how ofter the code will run:

https://github.com/scania-digital-design-system/tegel/assets/62651103/80734d31-179d-479b-961c-1e37f82c5c6e

Reply:
Good catch!